packages/server/src/middleware/selfhost.ts

```diff
@@ -1,7 +1,9 @@
-const env = require("../environment")
+import env from "../environment"
+import { BBContext } from "@budibase/types"
+
 // if added as a middleware will stop requests unless builder is in self host mode
 // or cloud is in self host
-module.exports = async (ctx, next) => {
+module.exports = async (ctx: BBContext, next: any) => {
```

Comment:
I was gonna say to use `export =` here so it can be imported by other TS files. Had a look and I can't find where this is used, just a test